contracts/StakeHub.sol

```diff
@@ -8,10 +8,8 @@ import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import "./SystemV2.sol";
 import "./extension/Protectable.sol";
 import "./interface/0.8.x/IBSCValidatorSet.sol";
```

Comment:
please use some static lint tool to detect if any dead code or useless fields/functions

Reply:
fixed